src/o365/yammer/commands/message/message-get.ts

```diff
@@ -0,0 +1,112 @@
+import commands from '../../commands';
+import GlobalOptions from '../../../../GlobalOptions';
+import {
+  CommandOption, CommandValidate
+} from '../../../../Command';
+import YammerCommand from "../../../base/YammerCommand";
+import request from '../../../../request';
+
+const vorpal: Vorpal = require('../../../../vorpal-init');
+
+interface CommandArgs {
+  options: Options;
+}
+
+interface Options extends GlobalOptions {
+  id: number;
+}
+
+class YammerMessageGetCommand extends YammerCommand {
+  public get name(): string {
+    return `${commands.YAMMER_MESSAGE_GET}`;
+  }
+
+  public get description(): string {
+    return 'Returns a Yammer message';
+  }
+
+  public getTelemetryProperties(args: CommandArgs): any {
```

Comment:
You can remove this method completely since it is supposed to capture telemetry on none mandatory options usage. Since `id` is mandatory there isn't a need to log in here because it will be used for sure if this command is used.